phpunit/block-supports/elements-test.php

```diff
@@ -72,7 +72,12 @@ public function test_elements_block_support_class( $color_settings, $elements_st
 			),
 		);
 
-		$actual = gutenberg_render_elements_support( $block_markup, $block );
+		// To ensure a consistent elements class name it is generated within a
+		// `render_block_data` filter and stored in the `className` attribute.
+		// As a result the block data needs to be passed through the same
+		// function for this test.
```

Comment:
```
/*
 * Multiline comments
 * use this format
 */
```

Per WP Coding standards for PHP and JS. https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#5-inline-comments

A common gotcha is to start wiht `/**` -- please don't as that's used by parsers looking for docblocks